frontend/src/app.rs

```diff
@@ -27,8 +27,9 @@ pub fn app() -> Html {
     // only do it once
     use_effect_with_deps(
         move |_| {
-            let x = backend::initialize();
-            console::log_1(&format!("{:?}", x).into());
+            spawn_local(async move {
+                let _ = crate::hooks::init_files().await;
```

Comment:
instead of this can we make the hook use_files() and make it call the use_future hook?

Reply:
ok